### PR TITLE
Eager register recently removed locations.

### PR DESCRIPTION
**Pull request by SergeyTeplyakov**

### CODE REVIEW

File: Public/Src/Cache/ContentStore/DistributedTest/ContentLocation/LocalLocationStoreDistributedContentTests.cs [hunk 1]
```diff
@@ -40,6 +40,17 @@
 
 namespace ContentStoreTest.Distributed.Sessions
 {
+    internal static class TransitioningContentLocationStoreExtensions
```
Comment (dannyvv) [inline Public/Src/Cache/ContentStore/DistributedTest/ContentLocation/LocalLocationStoreDistributedContentTests.cs]: >TransitioningContentLocationStoreExtensions [](start = 26, length = 43)

Dupe, already in seperate cs file...

**Comment by SergeyTeplyakov** [comment]
do /azp run "Primary BuildXL PR Validation"

**Comment by SergeyTeplyakov** [comment]
do /azp run Primary BuildXL PR Validation

**Pull request merged**